11/jdk/windows/Dockerfile.hotspot.nightly.full

```diff
@@ -19,16 +19,13 @@
 
 FROM mcr.microsoft.com/windows/servercore:ltsc2016
 
-ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en' LC_ALL='en_US.UTF-8'
-
-
 # $ProgressPreference: https://github.com/PowerShell/PowerShell/issues/2138#issuecomment-251261324
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
 ENV JAVA_VERSION jdk11u
 
-ENV JAVA_URL https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk11u-2019-06-11-10-43/OpenJDK11U-jdk_x64_windows_hotspot_2019-06-11-10-43.msi
-ENV JAVA_SHA256 518b3df42047385ba8cf3241696ef942eb4d1e36c87427212093c08c32d8d088
+ENV JAVA_URL https://github.com/AdoptOpenJDK/openjdk11-binaries/releases/download/jdk11u-2019-06-20-00-23/OpenJDK11U-jdk_x64_windows_hotspot_2019-06-20-00-23.msi
+ENV JAVA_SHA256 99e475db353ec098a90fe7280ef8a376ef6fde8c37f7feb7a7a3e037594dba21
 
```

Comment:
Both of these ENV's are not relevant to the user of the docker image and would be better if these are part of the subsequent `RUN` statement (As we do for the Linux platforms).

Reply:
done